src/Plugin.php

```diff
@@ -37,4 +39,19 @@ class Plugin extends BasePlugin
     public function routes($routes)
     {
     }
+
+    /**
+     * load WyriHaximus/TwigView plugin.
+     *
+     * @param \Cake\Core\PluginApplicationInterface $app The host application
+     * @return void
+     */
+    public function bootstrap(PluginApplicationInterface $app)
+    {
+        if (!CorePlugin::getCollection()->has('WyriHaximus/TwigView')) {
+            $app->addPlugin('WyriHaximus/TwigView', ['bootstrap' => false]);
+            $plugin = CorePlugin::getCollection()->get('WyriHaximus/TwigView');
+            $plugin->bootstrap($app);
```

Comment:
Why are you disabling the bootstrap hook when calling addPlugin() but then calling it here?

Reply:
just for ensuring that `bootstrap()` calls correctly 
I had a problem with pluginbootstrap here https://github.com/cakephp/cakephp/pull/12245/commits/cbe166dbbc
and we can not calls `pluginBootstrap();` again to cover new plugins

Reply:
Ok. Previously when I did testing the iterator implementation in PluginCollection allowed newly added plugins to be part of the loop without requiring additional loops. Perhaps I broke something in the meantime.